README.md

```diff
@@ -66,22 +68,36 @@ To learn how this team works together read [HOW_WE_WORK](./docs/HOW_WE_WORK.md)
     └── src_generate.go
 ```
 
-## Contributing
 
-If you plan to contribute code, make sure to install the [pre-commit](https://github.com/pre-commit/pre-commit) tool, which manages our pre-commit hooks for things like linters, go fmt, go vet, etc.
+## Team
 
-We provide a `Makefile` rule to facilitate the setup:
+The current TestGround Team is composed of:
+
+- @raulk - Lead Architect, Engineer, Developer
+- @daviddias - Engineer, Developer, acting as interim PM for the project
+- @nonsense - Engineer, Developer, TestGround as a Service / Infrastructure Lead
+- @jimpick - Engineer, Developer
+- @stebalien - Engineer, Developer
+- @hacdias - Engineer, Developer
+- you! Yes, you can contribute as well, however, do understand that this is a brand new and fast moving project and so contributing might require extra time to onboard
+
+To learn how this team works together read [HOW_WE_WORK](./docs/HOW_WE_WORK.md)
 
 ```sh
 $ make pre-commit
 ```
 
+## Contributing
+
+If you plan to contribute code, make sure to install the [pre-commit](https://github.com/pre-commit/pre-commit) tool, which manages our pre-commit hooks for things like linters, go fmt, go vet, etc.
+
 [![](https://cdn.rawgit.com/jbenet/contribute-ipfs-gif/master/img/contribute.gif)](https://github.com/ipfs/community/blob/master/CONTRIBUTING.md)
 
 This repository falls under the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
 
 You can contact us on the freenode #ipfs-dev channel or attend one of our [weekly calls](https://github.com/ipfs/team-mgmt/issues/674).
 
+
```

Comment:
README updates are good 👍🏾

Reply:
I don't consider the README complete, but wanted to at least add a structure where the top section describes `TestGround` to outsiders and has a good overview of the system.

What we had in `Description` read more of a history/background for the project, rather what `TestGround` actually is.